app-frontend/config/webpack/environments/production.js

```diff
@@ -17,6 +17,20 @@ module.exports = function (_path) {
             filename: '[name].[chunkhash].js'
         },
         plugins: [
+        // Return error code when build fails.

```

Comment:
Can we swap this inline code for this plugin? https://github.com/TiddoLangerak/webpack-fail-plugin

(Disappointing that something like this is necessary.)